Sources/SnowManLogic/View/View.swift

```diff
@@ -0,0 +1,28 @@
+public class View {
+	var ui: Receivable 
+	var guess = ""
```

Comment:
Yuck. this is only ever used in the `receiveGuess` method, might as well just bind the variable there.